doc/classes/AnimationTreePlayer.xml

```diff
@@ -199,6 +204,7 @@
 			<return type="NodePath">
 			</return>
 			<description>
+				Returns the path to the [AnimationPlayer] node that serves as a control interface for this [code]AnimationTreePlayer[/code].
```

Comment:
Maybe:
Returns the path to the [AnimationPlayer] node that is providing animations for this [code]AnimationTreePlayer[/code].